README.md

```diff
@@ -1,7 +1,127 @@
-MetSim
-======
+METSIM: Meteorology Simulator
+=============================
+| MetSim Links & Badges              |                                                                             |
+|------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Is the number of dash/pipes consistent? It doesn't look like this abides by the 80 characters to a line rule?